tests/tracer/test_trace_utils.py

```diff
@@ -247,6 +233,19 @@ def test_whitelist_case_insensitive(self, span, integration_config):
         assert span.get_tag("http.response.headers.content-type") == "some;value"
 
 
+@pytest.fixture
```

Comment:
`span` and `int_config` fixtures are not used in TestHeaders

Reply:
It is probably better to keep these defined above to limit the number of changes to this test.

We also _typically_ define all our fixtures at the top of the file, so they were ok to stay where they were